extensions/geode-modules-session-internal/src/main/java/org/apache/geode/modules/session/internal/common/AbstractSessionCache.java

```diff
@@ -31,19 +29,19 @@
    * The sessionRegion is the <code>Region</code> that actually stores and replicates the
    * <code>Session</code>s.
    */
-  protected Region<String, HttpSession> sessionRegion;
+  Region<String, HttpSession> sessionRegion;
 
   /**
    * The operatingRegion is the <code>Region</code> used to do HTTP operations. if local cache is
    * enabled, then this will be the local <code>Region</code>; otherwise, it will be the session
    * <code>Region</code>.
    */
-  protected Region<String, HttpSession> operatingRegion;
+  Region<String, HttpSession> operatingRegion;
 
-  protected Map<CacheProperty, Object> properties =
-      new TypeAwareMap<CacheProperty, Object>(CacheProperty.class);
+  protected TypeAwareMap<CacheProperty, Object> properties =
```

Comment:
For this one, I think that Map is actually a better choice (though it doesn't have any significant impact here).  Usually you just want to reference the interface wherever possible, that way we aren't coupled with a specific Map implementation (TypeAwareMap here).  Maybe I'm overlooking some other part of the code where this change was required so we could call methods on TypeAwareMap that aren't present on Map though, which would be a legitimate reason to change it.

Reply:
Good catch. I didn't have any "valid" reason to change it, other than getting rid of the `unchecked assignment` warning.